README.md

```diff
@@ -20,15 +29,21 @@ unzip master.zip
 rm master.zip
 ```
 
+## Usage
+
+Create new Pimcore document pages via the document types that are offered by this bundle.
+
 ## Configuration
 
-The Bundle provides a handful of _simple_ Areabricks to create Presentations. When the Bundle is used alongside other Bundles it might lead to Areabrick name collisions or one is just not interested in using the _default_ Bricks. It is possible to disable the included Areabricks using the following configuration
+The bundle provides a handful of _simple_ Areabricks to create Presentations. When the bundle is used alongside other bundles it might lead to Areabrick name collisions or one is just not interested in using the _default_ Bricks. It is possible to disable the included Areabricks using the following configuration
```

Comment:
```suggestion
The bundle provides a handful of _simple_ areabricks for creating presentations. When the bundle is used together with other bundles, there may be collisions of areabrick names or you may simply not be interested in using the _default_ bricks. It is possible to disable the included areabricks with the following configuration
```

Reply:
I think we should discuss whether we should namespace the bricks to avoid collisions in the first place.